applications/asset_tracker_v2/tests/gps_module/src/gps_module_test.c

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <unity.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <mock_modules_common.h>
+#include <mock_event_manager.h>
+#include <drivers/mock_gps.h>
+#include "app_module_event.h"
```

Comment:
Could we separate includes included with `""` and `<>` with a newline? This is _typically_ how we do it for modules in ATv2.